app/views/ActionSheet.js

```diff
@@ -0,0 +1,181 @@
+import vibrate from '../utils/throttle';
+
+/*
+To use this emmit an event to LISTNER and pass an object with snapPoint and options as parameter
+
+example object = {
+	snapPoint: SNAP_POINT.HALF,
+	options: {}
+}
+
+option object that you pass should have
+1. lable
+2. hadler
+3. icon // if not provided gives an flag icon
+
+example optionObject : {
+	label: 'Permalink',
+	handler: this.dummy,
+	icon: 'permalink'
+}
+
+*/
+
+export const LISTNER = 'actionSheet';
+export const SNAP_PONITS = {
+	HIDE: 0,
+	SHORT: 1,
+	HALF: 2,
+	FULL: 3
+};
+
+const bottomSheetHeaderHeight = 25;
+const buttonPaddingsSize = verticalScale(25);
+const textSize = 13;
+
+const styles = StyleSheet.create({
+	panel: {
+		height: verticalScale(500),
+		padding: verticalScale(5),
+		backgroundColor: '#f3f3f3'
+	},
+	panelButton: {
+		padding: verticalScale(10),
+		backgroundColor: 'transparent',
+		borderBottomWidth: 1,
+		borderBottomColor: '#dadada',
+		flexDirection: 'row'
+	},
+	panelButtonIcon: {
+		paddingHorizontal: 5
+	},
+	header: {
+		backgroundColor: '#f3f3f3',
+		paddingTop: 15,
+		borderTopLeftRadius: 20,
+		borderTopRightRadius: 20
+	},
+	panelHeader: {
+		alignItems: 'center'
+	},
+	panelHandle: {
+		width: 40,
+		height: 8,
+		borderRadius: 4,
+		backgroundColor: '#dadada'
+	},
+	androidButtonView: {
+		borderBottomWidth: 1,
+		borderBottomColor: '#dadada'
+	}
+});
+
+export default class ActionSheet extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			options: [
+				{ label: 'Permalink', handler: this.dummy, icon: 'permalink' },
+				{ label: 'Copy', handler: this.dummy, icon: 'copy' },
+				{ label: 'Share', handler: this.dummy, icon: 'share' }
+			]
+		};
+		this.bottomSheetRef = React.createRef();
+	}
+
+	componentDidMount() {
+		EventEmitter.addEventListener(LISTNER, this.handleDisplay);
+	}
+
+	componentWillUnmount() {
+		EventEmitter.removeListener(LISTNER);
+	}
+
+	dummy = () => {}
+
+	handleDisplay = (args) => {
+		this.setState({ options: args.options });
+		this.bottomSheetRef.current.snapTo(args.snapPoint);
+		vibrate();
+	}
+
+	hideActionSheet = () => {
+		this.bottomSheetRef.current.snapTo(0);
+	}
+
+
+	renderHeader = () => (
+		<View style={styles.header}>
+			<View style={styles.panelHeader}>
+				<View style={styles.panelHandle} />
+			</View>
+		</View>
+	);
+
+	buttonIcon = (icon) => {
+		if (!icon) {
+			icon = 'flag';
+		}
+		return (
+			<CustomIcon name={icon} size={18} style={styles.panelButtonIcon} />
+		);
+	}
+
+	buttonText = label => (
+		<Text style={sharedStyles.textRegular}>{label}</Text>
+	)
+
+	renderInner = () => {
+		const { options } = this.state;
+		const height = options.length * buttonPaddingsSize + options.length * textSize + bottomSheetHeaderHeight;
+		return (
+			<View style={[styles.panel, { height }]}>
+				{options.map(option => (
+					isIOS
+						? (
+							<Touchable onPress={() => { this.hideActionSheet(); option.handler(); }} key={option.label}>
```

Comment:
Create it with the same layout of autocomplete container.
![image](https://user-images.githubusercontent.com/804994/60438551-4e01c400-9be7-11e9-8aaa-79924f1408d0.png)